composer.json

```diff
@@ -48,6 +48,7 @@
         "php-http/psr7-integration-tests": "^1.3",
         "phpunit/phpunit": "^9.5.28",
         "psalm/plugin-phpunit": "^0.18.4",
+        "rector/rector": "^0.17.0",
```

Comment:
This is something I'd really like to see in our CI, rather than in a dev-dependency.

That said, it is fine here too, **for now**

Reply:
There is no phar available for rector, but we could include it as a global dependency in CI, but that could lead to an introduced failure without any code changes in the repositories.

Reply:
@internalsystemerror I would make a separate `composer.lock` just for that, installed globally in our CI container.

Not a problem for now though: just raising the idea :-)

Related: https://github.com/laminas/laminas-ci-matrix-action/issues/131

Reply:
No please keep it coming, thats the point of this PR as a draft, to get feedback!

Reply:
Use pinned version 0.17.0 is better imo